zebROS_ws/src/behaviors/src/game_piece_path_gen.cpp

```diff
@@ -0,0 +1,137 @@
+#include <ros/ros.h>
+#include <cmath>
+#include <std_srvs/Trigger.h>
+#include <algorithm>
+#include "base_trajectory_msgs/GenerateSpline.h"
+#include "behavior_actions/DynamicPath.h"
+#include "behavior_actions/GamePiecePickup.h"
+#include "field_obj/Detection.h"
+#include "trajectory_msgs/JointTrajectoryPoint.h"
+
+// Create service clients
+ros::ServiceClient spline_gen_cli;
+ros::ServiceClient dynamic_path_cli;
+
+field_obj::Detection lastObjectDetection;
+
+// Callback function to retrieve the most recent object detection message
+void objectDetectCallback(field_obj::DetectionConstPtr msg)
+{
+	lastObjectDetection = *msg;
+}
+
+trajectory_msgs::JointTrajectoryPoint generateTrajectoryPoint(double x, double y, double rotation) {
+	trajectory_msgs::JointTrajectoryPoint point;
+	point.positions.resize(3);
+	point.positions[2] = rotation;
+	point.positions[0] = x;
+	point.positions[1] = y;
+	return point;
+}
+
+bool genPath(behavior_actions::GamePiecePickup::Request& req, behavior_actions::GamePiecePickup::Response& res)
+{
+	res.success = false; // Default to failure
+  base_trajectory_msgs::GenerateSpline spline_gen_srv;
```

Comment:
The indenting here seems like spaces and tabs from the new vs old code might have gotten interleaved.  Can you run a formatting pass on it using 
`astyle --options=/home/ubuntu/2020Offseason/astyle.config foo.cpp`

Typically I wouldn't ask but since the file is new, let's start with a consistent format and go from there.

Reply:
Ok, thanks for reviewing this! I just ran astyle and will commit changes once I finish looking through the review.